dataprofiler/profilers/utils.py

```diff
@@ -430,11 +429,19 @@ def find_diff_of_lists_and_sets(stat1, stat2):
         pass
     elif stat1 is None or stat2 is None:
         diff = [stat1, stat2]
-    elif set(stat1) != set(stat2):
-        unique1 = [element for element in stat1 if element not in stat2]
-        shared = [element for element in stat1 if element in stat2]
-        unique2 = [element for element in stat2 if element not in stat1]
-        diff = [unique1, shared, unique2]
+    elif set(stat1) != set(stat2) or len(stat1) != len(stat2):
+        temp_stat1 = list(copy.deepcopy(stat1))
```

Comment:
added deepcopy but then assigned to list because testing output expects lists not sets